ecosystem/sep-0031.md

```diff
@@ -346,14 +346,14 @@ Name | Type | Description
 `amount_in` | string | (optional) The amount received or to be received by the Receiving Anchor. Up to 7 decimals are permitted. Excludes any fees charged after Receiving Anchor receives the funds.
 `amount_out` | string | (optional) The amount sent by the Rending Anchor to Receiving Client. Up to 7 decimals are permitted.
 `amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor.
-`stellar_account_id` | string | The Stellar account to send the payment to.
+`stellar_account_id` | string | The Receiving Anchor's Stellar account the Sending Anchor will be making the payment to.
 `stellar_memo_type` | string | The type of memo to attach to the Stellar payment: `text`, `hash`, or `id`.
 `stellar_memo` | string | The memo to attach to the Stellar payment.
 `started_at` | UTC ISO 8601 string | (optional) Start date and time of transaction.
 `completed_at` | UTC ISO 8601 string | (optional) Completion date and time of transaction.
-`stellar_transaction_id` | string | (optional) The transaction_id on Stellar network of the transfer that initiated the payment.
+`stellar_transaction_id` | string | (optional) The transaction_id on Stellar network of the transfer that initiated the payment from the Sending Anchor to the Receiving Anchor.
 `external_transaction_id` | string | (optional) The ID of transaction on external network that completes the payment into the receivers account.
-`refunded` | boolean | (optional) Should be true if the transaction was refunded. Not including this field means the transaction was not refunded.
+`refunded` | boolean | (optional) Should be true if the transaction was refunded. Refunded transactions should also have their `status` set to `error`. Not including this field means the transaction was not refunded.
```

Comment:
❔ Why do refund transactions have a status of error?

Reply:
There isn't a better status for it. In retrospect we should have used a `refunded` status instead of a boolean here.